.github/workflows/build-test.yml

```diff
@@ -348,23 +348,15 @@ jobs:
         with:
           python-version: ${{matrix.pythonVersion}}
 
-      - name: Cache conda on linux
-        uses: actions/cache@v2
+      - name: Install Conda environment from environment.yml
+        uses: mamba-org/provision-with-micromamba@main
```

Comment:
We want CI to be repeatable so pointing at main isn't a good idea. Also branches aren't the best as they're mutable, it's best to point at commits for third party actions.

Reply:
Done